evalml/data_checks/no_variance_data_check.py

```diff
@@ -13,7 +13,13 @@
 
 
 class NoVarianceDataCheck(DataCheck):
-    """Check if the target or any of the features have no variance."""
+    """Check if the target or any of the features have no variance.
+
+    Arguments:
+        count_nan_as_value (bool): If True, missing values will be counted as their own unique value.
+            If set to True, a feature that has one unique value and all other data is missing, a
```

Comment:
This second sentence is a little confusing. 
`If set to True, a feature that has one unique value and all other data is missing, a DataCheckWarning will be returned instead of an error.` -> `Additionally, if true, will return a DataCheckWarning instead of an error if the feature has mostly missing data and only one unique value.`?